app-frontend/src/app/components/tools/toolCreateModal/toolCreateModal.html

```diff
@@ -0,0 +1,77 @@
+<div class="modal-scrollable-body modal-sidebar-header">
+  <div class="modal-header">
+    <button type="button" class="close" aria-label="Close" ng-click="$ctrl.dismiss()">
+      <span aria-hidden="true">&times;</span>
+    </button>
+    <h4 class="modal-title">
+      Create a Tool
+    </h4>
+  </div>
+
+  <!-- Body for DATASOURCE_SELECT-->
+  <div class="modal-body">
+    <div class="content">
+      <p>
+
+      </p>
+      <form>
+        <div class="form-group">
+          <label for="name">Tool Name</label>
+          <input id="name" type="text" class="form-control"
+                 placeholder="New tool name"
+                 ng-model="$ctrl.toolBuffer.title"
+                 ng-disabled="$ctrl.isProcessing">
+        </div>
+        <div class="form-group">
+          <label for="name">Tool Description</label>
+          <input id="name" type="text" class="form-control"
+                placeholder="New tool description"
+                ng-model="$ctrl.toolBuffer.description"
+                ng-disabled="$ctrl.isProcessing">
+        </div>
+        <div class="form-group">
+          <label for="name">Tool Definition</label>
+          <p>
+            Enter an algebraic expression that represents the operation the tool should compute. Here are some hints:
+            <ul>
+              <li><strong>Allowed Operations</strong> - +, -, *, /</li>
```

Comment:
This might be premature, but it'd be nice if there was a way to codify this such that it would auto-update as we add more symbols. So if there were a list somewhere of `const VALIDOPERATIONS = ['-', '+',...]` and then here we could `<span ng-repeat="symbol in VALIDOPERATIONS">{{symbol}} </span>`